tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -402,9 +402,13 @@
   "DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML": "appsec_blocked_template_html",
   "DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON": "appsec_blocked_template_json",
   "DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING": "appsec_auto_user_events_tracking",
+  "DD_APPSEC_RASP_ENABLED": "appsec_rasp_enabled",
```

Comment:
I believe a change is needed in dd-go for this, @anna-git knows what to do.

Reply:
We are safe as long as no new tracer version is released. If this PR gets approved, then, I can merge in dd-go before merging this one just to be sure that no tracer is released with these changes before dd-go.

Reply:
The corresponding dd-go Pr would be this one: https://github.com/DataDog/dd-go/pull/121291